decidim-accountability/app/views/decidim/accountability/results/_home_header.html.erb

```diff
@@ -16,10 +16,8 @@
           <div class="progress-figure">
             <%= display_percentage progress_calculator(current_scope, nil) %>
           </div>
-        <% end %>
-
-        <%= render partial: "decidim/accountability/shared/extra" %>
```

Comment:
Where did this go?

Reply:
This was a hack - the file is empty (I'm actually removing it in a further commit), and was getting overriden from Barcelona. I've overriden the whole file instead. We have to find a way to modify views in a better way - thinking about using something like https://github.com/spree/deface for that.